guake/boxes.py

```diff
@@ -381,9 +381,8 @@ def set_search(self, widget):
 
             # Set search regex on term
             self.searchstring = text
-            self.searchre = GLib.Regex(text, 0, 0)
-            term.search_set_gregex(self.searchre, 0)
-
+            self.searchre = Vte.Regex.new_for_search(text, -1, 1024)
```

Comment:
I believe the flags can be setup more properly, we can take the reference to:
https://github.com/gnome-terminator/terminator/blob/master/terminatorlib/regex.py#L20

And the PCRE2 flag bits can be found at:
https://vcs.pcre.org/pcre2/code/trunk/src/pcre2.h.in?view=markup

Setting with 0x400 only give us `PCRE2_MULTILINE`

Reply:
I don't know which additional bits should be set besides those for PCRE2_MULTILINE?

Reply:
Perhaps `Vte.REGEX_FLAGS_DEFAULT` should be apply:

```
>>> import gi
>>> gi.require_version('Vte', '2.91')  # vte-0.38 (gnome-3.14)
>>> from gi.repository import GLib, Vte
>>> hex(Vte.REGEX_FLAGS_DEFAULT)
'0x40180000'
```

Including:

* #define PCRE2_NO_UTF_CHECK        0x40000000u
* #define PCRE2_UTF                 0x00080000u  /* C J M D */
* #define PCRE2_NEVER_BACKSLASH_C   0x00100000u  /* C       */